### PR TITLE
docs: 2024-10-25 changelog

### PR DESCRIPTION
**Pull request by bgrenon**
https://neon-next-git-bgrenon-2024-10-25-changelog-neondatabase.vercel.app/docs/changelog

### CODE REVIEW

File: content/changelog/2024-10-25.md [hunk 1]
```diff
@@ -0,0 +1,85 @@
...
+| pgvector      | 7.2         | 7.4         |
+| pgtap         | 1.2.0       | 1.3.3       |
+| plpgsql_check | 2.5.3       | 2.7.11      |
+| wal2json      | 2.5         | 2.6         |
+
+For a complete list of Postgres extensions supported by Neon and upgrade instructions, see [Postgres extensions](/docs/extensions/pg-extensions).
+
+### Easier User to Org account conversion
+
+We’ve introduced a new way to convert personal Neon accounts to organization accounts. Previously, this process required setting up a new organization, transferring projects, and entering billing details again. You can now convert your user account to an organization with a single action. This will transfer all of your projects and billing to the new organization instantly, with no service disruption and no changes to your connections. Your personal account will automatically switch to the Free Plan after conversion.
```
Comment (PlayLikeNeverB4) [inline content/changelog/2024-10-25.md:44]: nit: I've seen that we mix `’` and `'`, so not sure which the intended one is.

**Review by PlayLikeNeverB4 — APPROVED** [review]
Org change looks good

**Review by bgrenon — COMMENTED** [review]
LGTM

**Review by duskpoet — APPROVED** [review]

File: content/changelog/2024-10-25.md [hunk 2]
```diff
@@ -0,0 +1,85 @@
...
+- **Neon API changes**
+
+  - Updated the [Create Project](https://api-docs.neon.tech/reference/createproject) API to return a 404 error instead of a 500 error when an invalid region is specified.
+  - Updated the `project_id` field for the [Get consumption metrics for each project](https://console.neon.tech/api/v2/consumption_history/projects) API. You can now specify project IDs as a comma-separated list in addition to an array of parameter values. This provides more flexibility when filtering responses by project. If omitted, the response will include all projects.
+    - As an array: `project_ids=cold-poetry-09157238&project_ids=quiet-snow-71788278`
+    - As a comma-separated list: `project_ids=cold-poetry-09157238,quiet-snow-71788278`
+
+- **More local disk space for Neon computes**
+
+  We increased the local disk space allocation for Neon computes, which will now receive 15 GiB x maximum vCPU setting with a minimum of 20 GiB. This change ensures optimal handling of temporary data, query operations, and maintenance tasks in Postgres.
```
[the remark this answers is not in the document]
Comment (danieltprice) [reply content/changelog/2024-10-25.md:97]: If we're still working on it, I'll leave this one out.

**Pull request merged**